### ISSUE TITLE
Feature: Copy component REACT

### ISSUE THREAD

**Issue by WickedInvi** [saadeghi/daisyui#1034]
I am working on adding a react tab for all components but I am running into an issue with the styling with prism.

![image](https://user-images.githubusercontent.com/49752323/185144991-e1d69982-fa3c-4a1b-95e5-36a7271ca9b6.png)
Since react style takes {{ }} the highlighting is off as it does not seem to supper jsx.

I cannot seem to find out how to add support for it. Is it possible? Can you point me in the right direction?

**Comment by saadeghi** [comment]
I hardcoded the `language` to `html` on [this component](https://github.com/saadeghi/daisyui/blob/0695e3be3a4a1aaca08a748e27ec7760b94c144d/src/docs/src/components/Component.svelte#L78) because I just needed HTML back then. I think if you pass it as a prop and change it to `jsx` that would fix it.

Let me know if you need anything

**Comment by WickedInvi** [comment]
Sorry, I should have mentioned that I already tried passing `jsx` as language and it returns undefined. I've checked the list of languages and `jsx` is supported.

![image](https://user-images.githubusercontent.com/49752323/185298132-b598c708-bf7a-4514-9eca-a8356d4f9d60.png)
![image](https://user-images.githubusercontent.com/49752323/185298345-6bb3bd24-4351-4ed8-9471-24de3e0ba03d.png)

I've tested with `javascript` and it takes in and does not show undefined but is not what is required. But this means that it does take other languages

![image](https://user-images.githubusercontent.com/49752323/185302178-d518b624-ce8e-4ce8-84bc-82785d21b80d.png)

Does anything need to change here?  I am not very familiar with Prism or Svelte.
```svelte
<pre class="language-{language}" command-line data-output="2-17"><code class="language-{language}">{#if language === "none"}{formattedCode}{:else}{@html formattedCode}{/if}</code></pre>
```

I was looking at Prism and seems to me that languages might need to be defined. Maybe the package does not contain the language? Sorry, I am shooting in the dark trying to see what hits.
